front/src/applications/stdcmV2/components/StdcmConsist.tsx

```diff
@@ -35,9 +37,31 @@ const StdcmConsist = ({ setCurrentSimulationInputs, disabled = false }: StdcmCon
   const { speedLimitByTag, speedLimitsByTags, dispatchUpdateSpeedLimitByTag } =
     useStoreDataForSpeedLimitByTagSelector();
 
-  const { updateRollingStockID } = useOsrdConfActions() as StdcmConfSliceActions;
+  const { updateRollingStockID, updateTotalMass, updateTotalLength, updateMaxSpeed } =
+    useOsrdConfActions() as StdcmConfSliceActions;
   const dispatch = useAppDispatch();
 
+  const { getTotalMass, getTotalLength, getMaxSpeed } =
+    useOsrdConfSelectors() as StdcmConfSelectors;
+
+  const totalMass = useSelector(getTotalMass);
+  const onTotalMassChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const totalMassValue = Number(e.target.value);
+    dispatch(updateTotalMass(totalMassValue === 0 ? undefined : totalMassValue));
+  };
+
+  const totalLength = useSelector(getTotalLength);
+  const onTotalLengthChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const totalLengthValue = Number(e.target.value);
+    dispatch(updateTotalLength(totalLengthValue === 0 ? undefined : totalLengthValue));
+  };
+
+  const maxSpeed = useSelector(getMaxSpeed);
+  const onMaxSpeedChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const totalMaxSpeed = Number(e.target.value);
+    dispatch(updateMaxSpeed(totalMaxSpeed === 0 ? undefined : totalMaxSpeed));
+  };
```

Comment:
Maybe we could created a custom hook `useStdcmConsist` that returns all these values ? (at least for selector and actions)